composer.json

```diff
@@ -199,7 +211,12 @@
         },
         {
             "type": "path",
-            "url": "src/Symfony/Component/Runtime"
+            "url": "src/Symfony/Component/Runtime",
+            "options": {
+                "versions": {
+                    "symfony/runtime": "6.2.x-dev"
```

Comment:
I had to add this otherwise Composer didn't found a matching version for `symfony/runtime`.

Reply:
Does it help to install git into the image?

Reply:
Thanks for your answer, sadly adding `git` doesn't fix the issue. I saw that in #44678, I tried some days ago and just now, it still fails.

Reply:
Switching to Debian fix the issue, it will result in a bigger image than Alpine but it avoids modifying the `repositories` part of `composer.json`.